app/assets/stylesheets/components/_clockdiv.scss

```diff
@@ -0,0 +1,36 @@
+body{
```

Comment:
This will change the body tags for the entire project and could cause unpredictable behavior later on - probably safer to just give this a class or an id